### DISCUSSION TITLE
There is a chance to make it Laravel 10 comptatible?

### DISCUSSION THREAD

**Discussion by EhtuCom** [zofe/rapyd-livewire#3] [Ideas]
That's what I get when trying to install it in Laravel 10:

`Info from https://repo.packagist.org: #StandWithUkraine
./composer.json has been updated
Running composer update zofe/rapyd-livewire
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - zofe/rapyd-livewire 0.8.1 requires illuminate/contracts ^8.0 -> found illuminate/contracts[v8.0.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - zofe/rapyd-livewire[0.8.2, ..., 0.8.5] require illuminate/contracts ^8.27 -> found illuminate/contracts[v8.27.0, ..., v8.83.27] but these were not loaded, likely because it conflicts with another require.
    - zofe/rapyd-livewire[0.8.6, ..., 0.8.14] require illuminate/contracts ^8.27|^9.0 -> found illuminate/contracts[v8.27.0, ..., v8.83.27, v9.0.0, ..., v9.52.4] but these were not loaded, likely because it conflicts with another require.
    - zofe/rapyd-livewire[0.8.15, ..., 0.8.28] require illuminate/config ^8.65|^9.0 -> found illuminate/config[v8.65.0, ..., v8.83.27, v9.0.0, ..., v9.52.4] but these were not loaded, likely because it conflicts with another require.
    - Root composer.json requires zofe/rapyd-livewire * -> satisfiable by zofe/rapyd-livewire[0.8.1, ..., 0.8.28].

Installation failed, reverting ./composer.json and ./composer.lock to their original content.
`
I will like to help but with packages and composer I'm really a null :S

Thank you in advance,
David Lyons

**Comment by zofe** [comment]
I'll work for this soon, i hope this weekend

**Reply by EhtuCom** [reply]
Great! Thank you! Your packages are amazing!

**Reply by zofe** [reply]
shuld be installable now on a laravel 10 since 0.8.29 version.

**Reply by EhtuCom** [reply]
Wow, the upgrade rate to Laravel 10 was lightning fast! I've already installed it and am eager to start using it. Your update process was remarkably swift, and I'm truly grateful for that. Thank you so much!

![image](https://user-images.githubusercontent.com/4345616/221231499-2a18b8c9-487b-4301-b810-3a59fda28bfc.png)
